### DISCUSSION TITLE
AR model and UI difference in android browsers

### DISCUSSION THREAD

**Discussion by NEX-Reality** [google/model-viewer#4394] [Q&A]
It was surprising to see the AR result of different mobile browsers (The test was conducted only in Android). Any idea what is causing this difference and is there a way to get the result consistent in different browsers?

**Main observations**
- Firefox and Tor browser are not even showing the AR button in WebXR
- Opera and Edge are showing different UI with appearances different than WebXR (Moslty HDRI is using)

**Least requirements**

- Can we get the scale percentage in the simple AR UI? (Only with 'close' button)
- Even if the UI is different in AR, is there a way to match the appearance of the model like the WebXR?

 **BROWSER**     | **AR BUTTON VISIBLE** | **MODEL APPEARANCE SAME AS WEBXR** 
-------------------|----------------------------|---------------------------------
 **CHROME**      | YES                   | YES                             
 **FIREFOX**       | NO                    | --                              
 **BRAVE**          | YES                   | YES                             
 **OPERA**         | YES                   | NO                              
 **EDGE**            | YES                   | NO                              
 **TOR BROWSER** | NO                    | --                              

![chrome](https://github.com/google/model-viewer/assets/136175916/e424fb29-b3c3-4ea5-8a64-3ce1247f89fe)
![Opera](https://github.com/google/model-viewer/assets/136175916/f14d98cf-57c9-4d85-8f4d-bb097103d251)

**Comment by elalish** [comment]
The first image is WebXR mode, the second image is SceneViewer mode. If you only want webXR, you can use `ar-modes="webxr"`, but then the browsers that don't support WebXR will show no AR button instead of falling back to Scene Viewer. Unfortunately AR is not universally supported across browsers yet.
